x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterAsyncEnrichStopIT.java

```diff
@@ -91,6 +94,13 @@ public void testEnrichAfterStop() throws Exception {
         // Run the stop request
         var stopRequest = new AsyncStopRequest(asyncExecutionId);
         var stopAction = client().execute(EsqlAsyncStopAction.INSTANCE, stopRequest);
+        // wait until reduce tasks are gone
```

Comment:
I think we also need to wait for the cluster-level reduction task started before calling stop.